src/pages/Home/Home.js

```diff
@@ -1,39 +1,178 @@
-import React from 'react';
-import './Home.css';
+import React, { useState, useEffect } from 'react';
+
 import { Link } from 'react-router-dom';
-import localIcon from '../../pics/LogoInverted.svg';
+
 import Map from './Map';
+
+import {
+    MainContainer,
+    Wrapper,
+    Title,
+    Paragraph,
+    StyledContainer,
+    StyledColumn,
+    SignUpButton,
+} from './Home.style';
+
+import './Home.css';
+
+import SignUpIcon from '../../assets/SignUpIcon.png';
+import InviteIcon from '../../assets/InviteIcon.png';
+import ConnectIcon from '../../assets/ConnectIcon.png';
+import localIcon from '../../pics/LogoInverted.svg';
+
+import { Row, Column } from '../../utils/components/Grid';
+import HomeCard from '../../components/HomeCard/home-card.component';
 import Button from '../../components/UI/Button/Button';
 
+import useWindowSize from '../../utils/effects/useWindowSize.effect';
+
 const Home = user => {
-    return true ? (
+    const size = useWindowSize();
+
+    const [fontSize, setFontSize] = useState(1);
+
+    useEffect(() => {
```

Comment:
Yes, that's correct. This is the media query implementation using React hooks.

I used the hooks because I wanted to make the font size adjust instantly whenever the screen size changes.

In this scenario, I consider this approach to be better than media queries because:
1. It separates the concerns, i.e: if you wanted to add something in the future or change the font sizes, it's very easy to do so. You don't need to look through a big CSS file
2. I only used it for font-sizes so I thought using Hooks would make more sense.
3. Reusable. You can use this hook in other components.

PS: To be honest, I've been doing a course about React lately, and I wanted to practice using Hooks more.

You can use media queries if you want.

Reply:
I would argue, that developers who don't know react hooks, this would look confusing compared to media queries. The css files are not really that big. Media queries can also be reused. However, there is no point in arguing. Your way is also valid.